src/main/java/com/cedarsoftware/util/io/JsonReader.java

```diff
@@ -57,9 +57,12 @@
     public static final String USE_MAPS = "USE_MAPS";                   // If set, the read-in JSON will be turned into a Map of Maps (JsonObject) representation
     public static final String UNKNOWN_OBJECT = "UNKNOWN_OBJECT";       // What to do when an object is found and 'type' cannot be determined.
     public static final String JSON_READER = "JSON_READER";             // Pointer to 'this' (automatically placed in the Map)
+    public static final String OBJECT_RESOLVER = "OBJECT_RESOLVER";     // Pointer to the current ObjectResolver (automatically placed in the Map)
```

Comment:
I don't see where this object resolver is used in the PR, this seems to be just set and not used.